Sources/OAuthSwiftCredential.swift

```diff
@@ -16,24 +16,24 @@ public protocol OAuthSwiftCredentialHeadersFactory {
 open class OAuthSwiftCredential: NSObject, NSCoding {
 
     public enum Version {
-        case oauth1, oauth2
+        case oauth1, oauth1RSA, oauth2
```

Comment:
After merging I will do some change, try to remove this case oauth1RSA
by cutting the link between version and signature or use oauth1(SignatureMethod)